core/services/keystore/keys/ocr2key/terra_key_bundle.go

```diff
@@ -44,7 +44,10 @@ func newTerraKeyBundleFrom(onchainSigningKeyMaterial, onchainEncryptionKeyMateri
 	if err != nil {
 		return nil, err
 	}
-	terraKeyRing := newTerraKeyring()
+	terraKeyRing, err := newTerraKeyring(onchainSigningKeyMaterial)
```

Comment:
not introduced in this PR but I noticed that newOffchainKeyring(onchainSigningKeyMaterial, onchainEncryptionKeyMaterial) doesn't actually use offchainKeyMaterial? Maybe cleanest would be to just change the API to newTerraKeyBundleFrom(randReader io.Reader) since we only ever pass a single reader

Reply:
yeah looks like none of the ocr2 keys use that param. We could definitely clean it up. Maybe a follow up PR?